packages/core/src/ui/MapPreview/MapPreview.tsx

```
@@ -0,0 +1,480 @@
+const BASE_CSS = `
+.maplibregl-map{overflow:hidden;position:relative;-webkit-tap-highlight-color:rgba(0,0,0,0)}
+.maplibregl-canvas-container{height:100%;width:100%}
+.maplibregl-canvas{left:0;position:absolute;top:0}
+.maplibregl-map:fullscreen{height:100%;width:100%}
+.maplibregl-canvas-container.maplibregl-interactive{cursor:grab;user-select:none}
+.maplibregl-canvas-container.maplibregl-interactive:active{cursor:grabbing}
+`;
+
+function injectBaseCss() {
+  const id = "portaljs-map-preview-css";
+  if (document.getElementById(id)) return;
```

ESLint

[error] 'document' is not defined.

(no-undef)